src/platform/storm/src/flash_dummy.rs

```diff
@@ -43,7 +43,7 @@ static mut FLASH_CLIENT: FlashClient = FlashClient {
 const MAX_PAGE_NUM: i32 = 80;   // Page to go up to
 
 impl Client for FlashClient {
-    fn command_complete(&self, error: Error) {
+    fn command_complete(&self, _error: Error) {
```

Comment:
Is this possibly a bad / dangerous "fix" for the warning?

Shouldn't the client check to make sure there wasn't an error?